### ISSUE TITLE
Zeebe Test engine should start on a different port

### ISSUE THREAD

**Issue by ingorichtsmeier** [camunda/zeebe-process-test#366]
**Description**

When I started a local docker-compose for dev, I cannot start the tests and get an error message: `Port 26500 already in use`.

It's is very inconvenient for a process automation developer to stop the runtime, run the tests, start the runtime, do the integration test, develop next feature...

The Zeebe engine started by the tests should use a port like 26499, best would to make this configurable.

**Comment by berndruecker** [comment]
I agree (I also did this in spring-zeebe: https://github.com/camunda-community-hub/spring-zeebe/blob/master/test/embedded/src/main/java/io/camunda/zeebe/spring/test/ZeebeTestClientSpringConfiguration.java#L27)

**Comment by pihme** [comment]
Partially implemented: default port is now a different one.

Configuration options not yet implemented.

**Comment by pihme** [comment]
That would work, too. Let's see what Remco prefers when he is back from the retreat

**Comment by berndruecker** [comment]
I quickly asked Remco - and he is OK with a random port
